storage/memory/log_storage.go

```diff
@@ -125,8 +125,18 @@ func (t *readOnlyLogTX) GetActiveLogIDs(ctx context.Context) ([]int64, error) {
 	defer t.ms.mu.RUnlock()
 
 	ret := make([]int64, 0, len(t.ms.trees))
-	for k := range t.ms.trees {
-		ret = append(ret, k)
+	for id, tree := range t.ms.trees {
+		if tree.meta.GetDeleted() {
```

Comment:
(nit) nicer to be consistent about using `meta.Field` vs `meta.GetField()`

Reply:
Done.